tests/x11/pidgin/prep_pidgin.pm

```diff
@@ -26,9 +26,13 @@ sub run {
     x11_start_program('pidgin');
     send_key "alt-c";
 
-    # pidgin main winodow is hidden in tray at first run
-    # need to show up the main window
-    if (sle_version_at_least('12-SP2')) {
+    # pidgin main window is hidden in tray at first run
+    # need to show up the main window (12-SP2 and SP3)
+    # the main window is shown correctly in SLE15
+    if (is_sle('>=15')) {
+        wait_still_screen;
+    }
+    elsif (is_sle('>=12-sp2')) {
```

Comment:
This could be also just
```
if (is_sle('>=12-sp2')) {
     hold_key "ctrl-alt";
     send_key "tab";
}
wait_still_screen;
```

Reply:
`is_sle('<=12-sp3')`

Reply:
I would roll the dice to be same also on SP4 :yum:

Reply:
Actually, there are three branches in this if statement so it would still need some magic around.

Reply:
I really do not know, if the SP4 will be more like 12SP3 or 15 (ie. main window in the tray or not)

Reply:
true